platform/aeternity/api.go

```diff
@@ -0,0 +1,73 @@
+package aeternity
+
+import (
+	"encoding/base64"
+	"github.com/trustwallet/blockatlas/coin"
+	"net/http"
+	"strings"
+
+	"github.com/spf13/viper"
+	"github.com/trustwallet/blockatlas"
+)
+
+type Platform struct {
+	client Client
+}
+
+func (p *Platform) Init() error {
+	p.client.URL = viper.GetString("aeternity.api")
+	p.client.HTTPClient = http.DefaultClient
+	return nil
+}
+
+func (p *Platform) Coin() coin.Coin {
+	return coin.Coins[coin.AE]
+}
+
+func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
+	addressTxs, err := p.client.GetTxs(address, 25)
```

Comment:
@eddywm Would be more sense not to pass limit value but and reuse default limit from https://github.com/TrustWallet/blockatlas/blob/master/tx.go#L20 when construct url. Currently we don't pass `limit` param for transaction fetching.